src/index.js

```diff
@@ -86,6 +105,7 @@ export default class AutoCheckElement extends HTMLElement {
         this.dispatchEvent(new CustomEvent('load'))
 
         const warning = data ? data.trim() : null
+        this.input.setCustomValidity('')
```

Comment:
I think we want to set validity only if the element has a `required` attribute. This will prevent interfering with other validity checks that may be tied to this input.